hack/unit_test.sh

```diff
@@ -1,12 +1,6 @@
 #!/usr/bin/env bash
 
 set -e
-echo "" > coverage.txt
 
-for d in $(go list ./pkg/... ./cmd/...); do
-  go test -race -coverprofile=profile.out -covermode=atomic $d
-  if [ -f profile.out ]; then
-    cat profile.out >> coverage.txt
-    rm profile.out
-  fi
-done
+go test -race -coverprofile=coverage.txt -covermode=atomic \
```

Comment:
BTW, why not move this line to makefile?

Reply:
moved to makefile